examples/tutorials/api/estimators.py

```diff
@@ -0,0 +1,286 @@
+
+
+######################################################################
+# A note on the internal representation
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# The result contains a reference spectral model, which defines the spectral shape.
+# Typically, it is the best fit model:
+
+print(fp_result.reference_model)
+
+######################################################################
+# `~gammapy.estimators.FluxPoints` are the represented by the "norm" scaling factor with
+# respect to the reference model:
+
+fp_result.norm.plot()
+plt.show()
+
+######################################################################
+# Dataset specific quantities ("counts like")
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# While the flux estimate and associated errors are common to all datasets,
+# the result also stores some dataset specific quantities, which can be useful
+# for debugging.
+# Here we remind the user of the meaning of the forthcoming quantities:
+#
+# -  ``counts``: predicted counts from the null hypothesis.
+# -  ``npred``: predicted number of counts from best fit hypothesis.
+# -  ``npred_excess``: predicted number of excess counts from best fit hypothesis.
+#
+# The `~gammapy.maps.region.ndmap.RegionNDMap` allows for plotting of multidimensional data
+# as well, by specifying the primary ``axis_name``:
+
+
+fp_result.counts.plot(axis_name="energy")
+plt.show()
+
+######################################################################
+#
+fp_result.npred.plot(axis_name="energy")
+plt.show()
+
+######################################################################
+#
+fp_result.npred_excess.plot(axis_name="energy")
+plt.show()
+
+######################################################################
+# Table conversion
+# ~~~~~~~~~~~~~~~~
+#
+# Flux points can be converted to tables:
+#
+
+table = fp_result.to_table(sed_type="flux", format="gadf-sed")
+display(table)
+
+######################################################################
+#
+table = fp_result.to_table(sed_type="likelihood", format="gadf-sed", formatted=True)
+display(table)
+
+
+######################################################################
+# Common API
+# ----------
+# In `GAMMAPY_DATA` we have access to other `~gammapy.estimators.FluxPoints` objects
+# which have been created utilising the above method. Here we read the PKS 2155-304 light curve
+# and create a `~gammapy.estimators.FluxMaps` object and show the data structure of such objects.
+# We emphasize that these follow a very similar structure.
+#
+
+######################################################################
+# Load the light curve for the PKS 2155-304 as a `~gammapy.estimators.FluxPoints` object.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+
+lightcurve = FluxPoints.read(
+    "$GAMMAPY_DATA/estimators/pks2155_hess_lc/pks2155_hess_lc.fits", format="lightcurve"
+)
+
+display(lightcurve.available_quantities)
+
```

Comment:
Did we decide against showing the data structure of the `lightcurve` here?

Reply:
I believe so, but happy to adjust if you feel it is necessary!